TCMSApp/src/client/app/layout/shell.controller.js

```diff
@@ -27,9 +27,26 @@
 
         function hideSplash() {
             //Force a 1 second delay so we can see the splash.
-            $timeout(function() {
+            $timeout(function () {
                 $rootScope.showSplash = false;
             }, 1000);
         }
+
+        if (window.localStorage.getItem('authorized') == 'true') {
```

Comment:
maybe it is not the best Idea. if we have some auth service, it can expose some method like `isAuthorized` and we shall not  do `window.localStorage.getItem('authorized') == 'true'` here and there. But only `if authService.isAuthorized()`

Also we will win then, when we will decide to change the place where the auth info is stored (then we shall only change it in one place and `authService.isAuthorized()` will remain unchanged)